constants.go

```diff
@@ -616,10 +616,20 @@ const (
 	// PresetGroupAccessRoleName is a name of a preset role that allows
 	// access to all user groups.
 	PresetGroupAccessRoleName = "group-access"
+
+	// PresetAutomaticAccessApprovalRoleName names a preset role that may
+	// automatically approve any Role Access Request
+	PresetAutomaticAccessApprovalRoleName = "@teleport-access-approver"
```

Comment:
@tcsc @mdwn Why do we need a separate role for this? Can we just update `reviewer` role Mike's adding to allow reviewing everything? Otherwise the role is called "reviewer" but only allows reviewing app access requests which is confusing IMO. And we won't need a yet another builtin role.

Reply:
@r0mant: The role @mdwn is adding only allows reviewing access requests for the preset `access` role - this role allows reviewing access to _any_ role.

Reply:
Sorry - reading your comment for comprehension this time 🙄. 

Yes, but we probably want to try and make the role used by the approver bot user not editable, as a user could break the auto-approve functionality by editing the `approver` role. If we're happy hiding and/or making the `reviewer` role uneditable, I'm OK with that. Otherwise I'd prefer to have a separate role.

Reply:
But there's no change in your PR that makes this role uneditable either, right? That's my point - this is basically a duplicate of "reviewer" role so why not simplify things and just use it? We can always add another role later if it becomes a problem.